README.md

```diff
@@ -60,9 +63,39 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+yum -y install epel-release
+yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
+    python2-six \
+    python-gobject-base \
+    python-jsonschema \
+    python-setuptools \
+    python-ipaddress
+echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+echo -e "[main]\nno-auto-default=*\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+systemctl restart NetworkManager
+systemctl restart openvswitch
+```
+
+### Install nmstate from PyPi
```

Comment:
Actually since we already suggest to enable epel, it would be even better to install it with

```yum --enable-repo epel-testing install nmstate``` - for this case a lot of the RPM dependencies do not need to be installed manually btw.

Reply:
I do not think it is nice to require `epel-testing` and I think the current steps are good enough.
This is also what we do in the containers, so these are "verified".

Reply:
Not sure why using pypi is better than using epel testing... There is also the COPR repo. IMHO PyPi is more for people who cannot use RPMs.